website/core/RemarkablePlugins.js

```diff
@@ -42,7 +42,7 @@ function htmlForCodeBlock(code) {
  * whenever an example that uses the SnackPlayer is updated with code
  * that requires a newer Expo SDK release.
  */
-const LatestSDKVersion = '26.0.0';
+const LatestSDKVersion = '31.0.0';
```

Comment:
we can remove this constant, I changed Snack to always use the latest SDK version in https://github.com/facebook/react-native-website/pull/1139